scripts/Inelastic/dos/load_euphonic.py

```diff
@@ -24,19 +25,27 @@ def euphonic_available():
     return True
 
 
-def get_data_with_euphonic(filename: str, cutoff: float = 20.):
+def euphonic_calculate_modes(filename: str, cutoff: float = 20.,
+                             gamma: bool = True,
+                             asr: Optional[str] = 'reciprocal'):
     """
     Read force constants file with Euphonic and sample frequencies/modes
 
     :param filename: Input data
     :param cutoff:
         Sampling density of Brillouin-zone. Specified as real-space length
-        cutoff.
+        cutoff in Angstrom.
+    :param gamma:
+        Shift sampling grid to include the Gamma-point.
+    :param asr:
```

Comment:
This is, consistently, the argument name used in Euphonic. It's not really a parameter one can play with without reading the documentation, because you have to choose from a set of accepted values. Still, it could be expanded to `acoustic_sum_rule` without much harm.

Reply:
that would be good, we could add a comment saying in Euphonic its called asr

Reply:
done